migrations/versions/0429_go_live_templates.py

```diff
@@ -0,0 +1,158 @@
+                name,
+                template_type,
+                created_at,
+                subject,
+                content,
+                archived,
+                service_id,
+                created_by_id,
+                version,
+                process_type,
+                hidden
+            )
+            VALUES (
+                '{continue_template_id}',
+                'Reminder: continue self-service go live journey (for organisation users)',
+                'email',
+                current_timestamp,
+                'Request to go live: ((service_name))',
+                '{continue_template_content}',
+                false,
+                '{current_app.config["NOTIFY_SERVICE_ID"]}',
+                '{current_app.config["NOTIFY_USER_ID"]}',
+                1,
+                'normal',
+                false
+            )
+            ON CONFLICT DO NOTHING
+            """
+        )
+
+    op.execute(
+        f"""
+        INSERT INTO template_redacted
+        (
+            template_id,
+            redact_personalisation,
+            updated_at,
+            updated_by_id
+        ) VALUES (
+            '{continue_template_id}',
+            false,
+            current_timestamp,
+            '{current_app.config["NOTIFY_USER_ID"]}'
+        )
+        ON CONFLICT DO NOTHING
+        """
+    )
+
+    for table_name in ("templates", "templates_history"):
+        op.execute(
+            f"""
+            INSERT INTO {table_name} (
+                id,
+                name,
+                template_type,
+                created_at,
+                subject,
+                content,
+                archived,
+                service_id,
+                created_by_id,
+                version,
+                process_type,
+                hidden
+            )
+            VALUES (
+                '{reject_template_id}',
+                'Reject self-service go live request',
+                'email',
+                current_timestamp,
+                'Your request to go live has been rejected',
+                '{reject_template_content}',
+                false,
+                '{current_app.config["NOTIFY_SERVICE_ID"]}',
+                '{current_app.config["NOTIFY_USER_ID"]}',
+                1,
+                'normal',
+                false
+            )
+            ON CONFLICT DO NOTHING
+            """
+        )
+
+    op.execute(
+        f"""
+        INSERT INTO template_redacted
+        (
+            template_id,
+            redact_personalisation,
+            updated_at,
+            updated_by_id
+        ) VALUES (
+            '{reject_template_id}',
+            false,
+            current_timestamp,
+            '{current_app.config["NOTIFY_USER_ID"]}'
+        )
+        ON CONFLICT DO NOTHING
+        """
+    )
+
+
+def downgrade():
+    # I don't consider there to be a safe downgrade path as other tables could theoretically have FK dependencies
```

Comment:
[comment, non-actionable] I agree once the feature is live, the downgrade would not be recommended to avoid losing notifications. I think downgrades are much more useful if something goes wrong during deployment. However, since this migration doesn't change schema, but just adds data, I don't think we need one.